### PR TITLE
Hosted docs - section on how to tell if scrapyd is doing work and how…

### PR DESCRIPTION
**Pull request by odscjames**
… to stop service safely

https://github.com/open-contracting/kingfisher-scrape/issues/94

### CODE REVIEW

**Comment by odscjames** [comment]
I know the "Are any spiders currently running?" section is small at the moment, but when we add the old spiders it will get bigger, so I want to get it into master now so i can build on it in a different pull request.

File: docs/use-hosted.rst [hunk 1]
```diff
@@ -51,3 +51,20 @@ Finally, you'll need to update scrapyd with the latest code. SSH into the server
...
+----------------------------------
+
+You may want to know this before doing server maintenance, stopping or starting Scrapyd, etc.
+
+Check the `Scrapyd web interface <http://scrape.ocdskingfisher.opendataservices.coop>`_ - click on `Jobs` and look in the `Running` section.
+
+Stopping and starting Scrapyd
+-----------------------------
+
+Log in to the server as the root user and run
```
Comment (robredpath) [inline docs/use-hosted.rst]: @odscjames under what circumstances might I want to do this?

**Comment by robredpath** [comment]
@odscjames I've approved, with one comment that can either be fixed now or in the next round of changes - up to you

**Comment by odscjames** [comment]
Will issue that - thanks

**Pull request merged**